src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise.component.html

```diff
@@ -1,5 +1,5 @@
 <div>
-    <h4>
+    <h4 *ngIf="isInExerciseGroup == null">
```

Comment:
Why not have this as a boolean get function for the exercise? You could move this as a helper function in the exercise model e.g `exercise.isExamExercise()`

Reply:
as discussed, attribute xxexercise.exerciseGroup can not be accessed in abstract class ExerciseComponent, where the loading of other exercise from the course (method load()) is called. This method shall be prevented for exam exercise, so a new attribute is needed.